.github/workflows/deploy_be_staging.yml

```diff
@@ -61,7 +61,10 @@ jobs:
           poetry run python3 data_pipeline/application.py score-full-run
       - name: Generate Score Post
         run: |
-          poetry run python3 data_pipeline/application.py generate-score-post -s aws
+          poetry run python3 data_pipeline/application.py generate-score-post
```

Comment:
What does removing `-s aws` here do?

Reply:
uses the data that was downloaded score-full-run, which saves us a download and speeds up the pipeline a little